doc/ref_model/chapters/appendix-a.md

```diff
@@ -16,19 +16,71 @@ This Appendix has two goals:
 1. Provide guidance to VNF or more generally Application vendors on how to consume CNTT Reference Model and Architectures
 2. Provide usable definitions of maturity levels for VNF software architecture between Physical-to-Virtual migration and “Cloud Native”.
 
-The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform.
+The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform. So, it will primarily look at Operational impact (in steps like Install, Configure, Validate, Operate) towards full lifecycle automation of Develop decisions made (in steps like Design, Simulate, Build, Bundle).
 
 <a name="A.2"></a>
 ## A.2 Intro and Terminology
-(Summary status and trends of ETSI NFV and Cloud. Decoupling application from platform. Terminology used/introduced.)
+(Summary status and trends of ETSI NFV and Cloud.)
+
+Terminology used or introduced:
+
+-	Decoupling, Loose Coupling = Loosely coupled system is one in which each of its components has, or makes use of, little or no knowledge of the definitions of other separate components. Loose coupling is the opposite of tight coupling. [1]
+
+-	Encapsulation = Restricting of direct access to some of an object's components. [2]
+
+-	Appliance deployment model = Application has tight coupling with underlying Platform. That application can be virtualized or containerized.
```

Comment:
Should this be, ".. cannot be virtualised or containerised"?

Reply:
> Should this be, ".. cannot be virtualised or containerised"?

That was not the intention. Rewrote it with "... even if...".